RELEASING.md

```diff
@@ -2,7 +2,13 @@
 
 The Debezium project uses Maven for its build system, relying up on the _release_ plugin to most of the work. This document describes the steps required to perform a release.
 
-# Update the changelog
+## Verify Jira issues
+All issues planned for this release must be resolved. If not they have to either be re-planned to another release or rejected. Use JQL query to find offending issues
+```
+project=DBZ AND fixVersion=<VERSION> AND status NOT IN ('Resolved', 'Closed')
```

Comment:
Hey @jpechane, just to be sure, the release job is doing this and aborts if there are any issues assigned to the fix version but in another state, correct?